tests/logger_structlog/test_metrics.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from newrelic.packages import six
+from newrelic.api.background_task import background_task
+from testing_support.fixtures import reset_core_stats_engine
+from testing_support.validators.validate_transaction_metrics import validate_transaction_metrics
+from testing_support.validators.validate_custom_metrics_outside_transaction import validate_custom_metrics_outside_transaction
+
+
+
+def exercise_logging(logger, structlog_caplog):
```

Comment:
Same thing here. Convert these functions into fixtures and if they are used in more than one module consider putting them in conftest.